phpBB/config/default/container/services.yml

```diff
@@ -21,6 +21,7 @@ imports:
     - { resource: services_notification.yml }
     - { resource: services_password.yml }
     - { resource: services_php.yml }
+    - { resource: services_post.yml }
```

Comment:
Hm, I wonder if posting instead of post would make more sense for future use?

Reply:
Ok, but it just a concept, to start moving querys to their corresponding folders, what do you think about that?
I saw chita is moving the querys to repositories, sould I start moving all the querys to helpers/repositorys?